src/parallel/MainSplitter.h

```diff
@@ -20,21 +20,21 @@
 class MainSplitter : public MainSolver {
 
 private:
-    inline bool isSplitTypeScatter() const &       { return dynamic_cast<Splitter&>(ts.solver).isSplitTypeScatter(); }
+    inline bool isSplitTypeScatter() const &       { return dynamic_cast<Splitter&>(*smt_solver).isSplitTypeScatter(); }
```

Comment:
Is there some diff between smt_solver and ts.solver?

Reply:
No, previously the Tseitin class hold a reference to the CoreSMTSolver, but part of the proposed change is to make the cnfization process independent of that.